cgc/triclustering.py

```diff
@@ -116,8 +120,53 @@ def run_with_threads(self, nthreads=1):
         self.results.write(filename=self.output_filename)
         return self.results
 
-    def run_with_dask(self, client=None):
-        raise NotImplementedError
+    def run_with_dask(self, client=None, low_memory=False):
```

Comment:
```suggestion
    def run_with_dask(self, client=None, low_memory=True):
```

Since we haven't implemented the non-low-memory yet, maybe it would be good to set low memory by default True?